src/libDirectoryService/ViewChangePreProcessing.cpp

```diff
@@ -261,6 +279,14 @@ void DirectoryService::ComputeNewCandidateLeader() {
         m_mediator.m_DSCommittee->at(m_viewChangeCounter).second;
   }
 
+  // Compute the CommitteeHash member of the BlockHeaderBase
+  CommitteeHash committeeHash;
+  if (!Messenger::GetDSCommitteeHash(*m_mediator.m_DSCommittee,
+                                     committeeHash)) {
+    LOG_EPOCH(WARNING, to_string(m_mediator.m_currentEpochNum).c_str(),
+              "Messenger::GetDSCommitteeHash failed.");
```

Comment:
return false here?

Reply:
Okay. for now this func only returns void. Will add boolean return to interrupt the function at higher layer.

Reply:
Fixed.